### DISCUSSION TITLE
Metadata Editing and Dropdown Menus

### DISCUSSION THREAD

**Discussion by Soeroah** [valinet/ExplorerPatcher#3695] [Q&A]
Hi,

I'm not sure if this is something ExplorerPatcher can help with, but recently my computer stopped showing me dropdowns of metadata history as I type text into the relevant metadata fields. For example, typing "Com" into the tag field used to bring up "Complete" but now it doesn't do that at all. Same for the Authors field.

Is there a possibility ExplorerPatcher could re-enable this functionality somehow?

Thank you

**Comment by pyrates999** [comment]
Uninstall EP, reboot, and see if it is still happening.

**Reply by Soeroah** [reply]
It does still happen, unfortunately.

I don't think the dropdown not appearing is a problem with ExplorerPatcher, I was just wondering if it was at all possible ExplorerPatcher might be able to find a workaround to force it to work since it fixes so many other issues in new Windows

**Reply by pyrates999** [reply]
Then this is not caused by EP.

try this to rebuild the search index: https://learn.microsoft.com/en-us/troubleshoot/windows-client/shell-experience/windows-search-performance-issues

**Reply by Soeroah** [reply]
Unfortunately that hasn't resolved it. I even reduced the amount of folders being indexed

**Reply by pyrates999** [reply]
Then you gotta wait until MS fixes it.
